Source: me#gmail.com
Upstream: Yes,r3988 (sigh, I found issue has been fixed by upstream as soon as this little patch was done)
Reason: duplicates in the file list are not allowed
--- supertuxkart-0.6.2/data/Makefile.am~	2009-10-15 22:26:03.743053073 +0800
+++ supertuxkart-0.6.2/data/Makefile.am	2009-10-15 22:27:46.162052764 +0800
@@ -6,7 +6,6 @@
 dist_pkgdata_DATA = \
  $(shell find $(srcdir) -name "*.data") \
  $(shell find $(srcdir) -name "*.items") \
- $(shell find $(srcdir) -name "*.projectile") \
  $(shell find $(srcdir) -name "*.cup") \
  $(shell find $(srcdir) -maxdepth 1 -name "*.challenge") \
  $(shell find $(srcdir) -name "*.collectable") \
